src/m_linemixing.cc

```diff
@@ -2964,7 +2964,7 @@ void abs_lines_per_bandSetLineMixingFromRelmat(
 }
 
 #include <Eigen/Eigenvalues>
-#include "Faddeeva.hh"
+#include "../3rdparty/Faddeeva/Faddeeva.hh"
 #include "physics_funcs.h"
```

Comment:
Please move the include statement to the top of the file.

Reply:
Also replace the relative include path by:

```cpp
include <Faddeeva/Faddeeva.hh>
```

This is better for treating Faddeeva as a 3rd party library.

Reply:
Done.